custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -265,11 +267,68 @@ def predict(self, data, model=None, **kwargs):
 
         return predictions
 
+    def try_to_fit_on_custom_wrapper(self, X, y, output_dir):
+        """
+        A user can surround an sklearn pipeline or estimator with the custom() function, importable
+        from drum, which will tag the object that is passed in with a to_upload variable.
```

Comment:
Probably not `to_upload` variable, but `is_custom`?

Reply:
everything in the file is custom.

Reply:
Quote: `which will tag the object that is passed in with a to_upload variable.`

What is **to_upload** variable?

Reply:
the to_upload variable is tagged by the custom() function which lets drum know to upload this estimator instead of another one in the file.

Reply:
Where is this variable in the code?

Reply:
ah right, I meant is_custom

Reply:
on third thought, I kinda want to change it back.